DNN Platform/Tests/DotNetNuke.Tests.Integration/Tests/Jwt/JwtAuthTest.cs

```diff
@@ -51,13 +51,13 @@ public class JwtAuthTest : IntegrationTestBase
         private readonly TimeSpan _timeout = TimeSpan.FromSeconds(30);
 #endif
 
-        private const string LoginQuery = "/DesktopModules/JwtAuth/API/mobile/login";
-        private const string LogoutQuery = "/DesktopModules/JwtAuth/API/mobile/logout";
-        public const string ExtendTokenQuery = "/DesktopModules/JwtAuth/API/mobile/extendtoken";
-        private const string TestGetQuery = "/DesktopModules/JwtAuth/API/mobile/testget";
-        private const string TestPostQuery = "/DesktopModules/JwtAuth/API/mobile/testpost";
-        private const string GetMonikerQuery = "/DesktopModules/web/API/mobilehelper/monikers?moduleList=";
-        private const string GetModuleDetailsQuery = "/DesktopModules/web/API/mobilehelper/moduledetails?moduleList=";
+        private const string LoginQuery = "/API/JwtAuth/Action/mobile/login";
+        private const string LogoutQuery = "/API/JwtAuth/Action/mobile/logout";
+        public const string ExtendTokenQuery = "/API/JwtAuth/Action/mobile/extendtoken";
+        private const string TestGetQuery = "/API/JwtAuth/Action/mobile/testget";
+        private const string TestPostQuery = "/API/JwtAuth/Action/mobile/testpost";
+        private const string GetMonikerQuery = "/API/web/Action/mobilehelper/monikers?moduleList=";
+        private const string GetModuleDetailsQuery = "/API/web/Action/mobilehelper/moduledetails?moduleList=";
 
```

Comment:
JWT is out there for a couple of releases (since the beginning of this year) and you can't say it is not being used until now. It is a must to guarantee backward compatibility since day 1 after releasing it to the public.